lib/turn.rb

```diff
@@ -0,0 +1,22 @@
+class Turn
+  attr_reader :guess , :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+
+  end
+
+  def correct?
+    guess == card.answer
```

Comment:
n+1 to the non-use of `@` when referencing instance variables. It might not seem like much now or in this example, but it will make a bigger difference when we start throwing around different types of scope.